cartography/data/jobs/cleanup/sumologic_import_cleanup.json

```diff
@@ -0,0 +1,10 @@
+{
+  "statements": [
+    {
+      "query": "WITH datetime()-duration('P7D') AS threshold MATCH (h:SumologicHost) WHERE h.lastupdated < threshold WITH h LIMIT $LIMIT_SIZE DETACH DELETE (h)",
```

Comment:
Can you tell us how this works?

Reply:
This is the same thing as before but cleaning only entries lastupdated older than 7 days as per https://github.com/lyft/cartography/issues/1015